### PR TITLE
Note EPEL, RHEL 6 Optional and min Puppet requirements

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: _includes/manuals/1.1/2_quickstart_guide.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ The Foreman installer is a collection of Puppet modules that installs everything
 Components include the Foreman web UI, Smart Proxy, Passenger (for the puppet master and Foreman itself), and optionally TFTP, DNS and DHCP servers.  It is configurable and the Puppet modules can be read or run in "no-op" mode to see what changes it will make.
 
 #### Supported platforms
-* RHEL derivatives (CentOS), 5 and 6
+* RHEL derivatives (CentOS), 5 and 6 (EPEL and RHEL 6 Optional required)
```
Comment (skottler) [inline _includes/manuals/1.1/2_quickstart_guide.md]: CentOS users might not know what the optional channel. What do you think about `EPEL is required for all EL systems. The "optional" channel should also be enabled on RHEL.`?

Comment (domcleal) [inline _includes/manuals/1.1/2_quickstart_guide.md]: I was trying to keep it short for the quickstart guide, compared to the multiple lines in the main install section.  How about, `EPEL required, plus subscribe to "optional" channel on RHEL 6` ?

Comment (skottler) [inline _includes/manuals/1.1/2_quickstart_guide.md]: Yep, that's great.

**Comment by domcleal** [comment]
Fixed, thanks for reviewing @skottler

**Comment by domcleal** [comment]
This was merged.

**Pull request closed without merging**